frontend/src/metabase/parameters/types.ts

```diff
@@ -0,0 +1,17 @@
+import Field from "metabase-lib/lib/metadata/Field";
```

Comment:
I'm keeping the types in this file separate from what's in `metabase-types` because they are ui-specifc and some of them are dependent on metabase-lib's `Field`